script/new-converter.js

```diff
@@ -0,0 +1,63 @@
+const { Command } = require("commander");
+const { promises: fs } = require("fs");
+const { upperFirst, camelCase } = require("lodash");
+
+(async () => {
+    const command = new Command()
+        .option("--eslint [eslint]", "name of the original ESLint rule")
+        .option("--tslint [tslint]", "name of the original TSLint rule");
+
+    const args = command.parse(process.argv).opts();
+
+    for (const arg of ["eslint", "tslint"]) {
+        if (!args[arg]) {
+            throw new Error(`Missing --${arg} option.`);
+        }
+    }
+
+    const tslintPascalCase = upperFirst(camelCase(args.tslint));
+    const plugins = args.eslint.includes("/")
+        ? `
+        plugins: ["${args.eslint.split("/")[0]}"],`
```

Comment:
Currently there is a plugin that has a prefix and it is not included in the plugin list in converters: `@typescript-eslint`. It is assumed that plugin is always installed I think?

Reply:
Ha, true. I was just thinking of the react-a11y ones!